api/circulation.py

```diff
@@ -259,13 +259,15 @@ class HoldInfo(CirculationInfo):
     """
 
     def __init__(self, collection, data_source_name, identifier_type,
-                 identifier, start_date, end_date, hold_position):
+                 identifier, start_date, end_date, hold_position,
+                 external_identifier=None):
         super(HoldInfo, self).__init__(
             collection, data_source_name, identifier_type, identifier
         )
         self.start_date = start_date
         self.end_date = end_date
         self.hold_position = hold_position
+        self.external_identifier = external_identifier
```

Comment:
Holds can now have an external_identifier. (Loans already could from my previous ODL work.) When a patron borrows or places a hold on a book from another circ manager, a URL for getting the status of the loan or hold is stored in external_identifier.